django.env.sample

```diff
@@ -3,8 +3,8 @@ SECRET=dummy
 ADMIN_USER=admin
 ADMIN_EMAIL=admin@example.com
 ADMIN_PASS=p@55w0rd1
+API_IDENTIFIER=https://chronoscio.org/api/
 
-AUTH0_DOMAIN={DOMAIN}
-AUTH0_CLIENT_ID={ID}
-AUTH0_CLIENT_SECRET={SECRET}
-API_IDENTIFIER={IDENTIFIER}
+AUTH0_DOMAIN={PROJECT.auth0.com}
+AUTH0_CLIENT_ID={Client ID}
+AUTH0_CLIENT_SECRET={Client Secret}
```

Comment:
Why was API identifier removed? Also, the sed regex in `.travis.yml` looks for specific values for these fields which it replaces with our secrets so that will have to be updated as well.

Reply:
I just separated it elsewhere in the file to look nicer. Will look into updating .travis.yml.

Reply:
Going to swap it back to {IDENTIFIER} I suppose for Travis to be the same, though it should not be necessary to include there? Why are some of these set with defaults in the sample env but others set with defaults in settings.py and/or .travis/yml? It seems kind of arbitrary. It feels more natural to be to just have one set of defaults and maaaaybe a second set of test-specific defaults but not 3 separate ones.

Reply:
Oh sorry, didn't catch that. It's an Auth0-specific thing so I think it'd be better placed in the Auth0 group. I agree about the defaults actually, let's just keep them all here for simplicity's sake. The only test specific defaults are so that all third party libs can use our access tokens and such which is why they aren't publicly displayed.